docs/docs/sourcing-from-wordpress.md

```diff
@@ -2,7 +2,119 @@
 title: Sourcing from WordPress
 ---
 
-This is a stub. Help our community expand it.
+This guide will walk you trough the process of using [Gatsby](https://www.gatsbyjs.org/) with [Wordpress Rest Api](https://developer.wordpress.org/rest-api/reference/).
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+WordPress is a free and open-source content management system (CMS). Let's say you have a site built with Wordpress and you want to pull the existing data into your static Gatsby site. You can do that with [gatsby-source-wordpress](https://www.gatsbyjs.org/packages/gatsby-source-wordpress/?=wordpress). Let's begin!
+
+_Note: this guide uses the `gatsby-starter-default` to provide you with a knowledge necessary to start working with Wordpress but if you get stuck at some point of the guide feel free to use
+[this example](https://github.com/gatsbyjs/gatsby/tree/master/examples/using-wordpress) to gain extra insights._
+
+## Setup
+
+###Bootstrapping a project
```

Comment:
Will you retitle this section "Quick start" and instead of giving instructions, say something like:
"This guide assumes that you have a Gatsby project set up. If you need to set up a project, head to the [Quick Start guide](/docs/), then come back."

I'm realizing this will save space and make sure we can maintain the Quick Start guide in just one place. Thanks! :)